openedx/core/djangoapps/schedules/admin.py

```diff
@@ -122,8 +122,8 @@ def choices(self, changelist):
 
 @admin.register(models.Schedule)
 class ScheduleAdmin(admin.ModelAdmin):
-    list_display = ('username', 'course_id', 'active', 'start', 'upgrade_deadline', 'experience_display')
-    list_display_links = ('start', 'upgrade_deadline', 'experience_display')
+    list_display = ('username', 'course_id', 'active', 'start', 'start_date', 'upgrade_deadline', 'experience_display')
+    list_display_links = ('start', 'start_date', 'upgrade_deadline', 'experience_display')
```

Comment:
This will only make the start_date column visible in the admin interface, but not make writes to start also write to start_date.  You need to either:

1. make writes to `start` from the django admin interface also go into `start_date`, probably by implementing a signal handler, or
2. determine if we can simply hope that nobody will use the django admin interface to change `start` while this is being deployed.

As-is, the two columns visible in the django admin will be confusing so I it should also be changed back to only displaying `start` since that is still the primary column.

Reply:
For my rename task in CreditRequirement, I am actually not going to implement any write abstraction layer for the django admin interface because I looked at the django admin log (`django_admin_log` table) and found that nobody was updating the table from django admin.  I also asked the team which seem likely to update the table, and they said they do not.

Reply:
I understand that this only makes this field available in admin only without allowing any writes from admin page. My understanding was that 

1. Schedules are only created when user enrolls in self-paced course otherwise schedule is not created
2. Also admin page is not doing any writes in schedules (**start**).
I will look and talk to people that are taking part in enrollment & schedule process.

Reply:
> Also admin page is not doing any writes in schedules (start).

Can you elaborate?  Are you saying that the admin page does not allow people to edit?

Reply:
@pwnage101 _Schedule_ is only created on post_save on CourseEnrollment model and not manually using django admin. 
As per discussion with Sustaining & Escalations team member, Altering from django admin page is not part of SOP.

Reply:
I'll use _start_ field until data migration is complete and will replace with _start_date_ afterwards.